src/app/interface/interface.ts

```diff
@@ -0,0 +1,21 @@
+export interface CovidInfo {
```

Comment:
I like this - very good. Keep the flow with the mandatory properties above the optional ones 👍 

FYI, good practice for naming interfaces is to put the capital I in front of the name (like: ICovidInfo)